config/plugins/interactive_environments/jupyter/config/jupyter.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE interactive_environment SYSTEM "../../interactive_environments.dtd">
+<interactive_environment name="Jupyter (Programming Environment)">
```

Comment:
Ahh, the good old naming debate. Let me add my 2c:
Omitting the `Python` keyword is one of the reasons they renamed to Jupyter, isn't it?